gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const gulp = require('gulp');
+const plumber = require('gulp-plumber');
+const postcss = require('gulp-postcss');
+const autoprefixer = require('autoprefixer');
+const groupmq = require('gulp-group-css-media-queries');
+
+const STYLE_SOURCES = [
+    './public/**/*.css',
+];
+
+function prefixStyles() {
+    return gulp.src(STYLE_SOURCES, { base: './' })
+        .pipe(plumber())
+        .pipe(postcss([
+            autoprefixer({
+                cascade: false
+            })
+        ]))
+        .pipe(groupmq())
+        .pipe(gulp.dest('.'));
+}
+exports.prefixStyles = prefixStyles;
+
+exports.default = gulp.series(prefixStyles);
```

Comment:
Even though I haven't set it up for PRs, this wouldn't pass linting. I can help with the code style when I pull down this repo next.